test/install.c

```diff
@@ -1124,6 +1141,20 @@ static void install_test_bundle(InstallFixture *fixture,
 	args->cleanup(args);
 }
 
+static void install_test_bundle_twice(InstallFixture *fixture,
+		gconstpointer user_data)
+{
+	/* First run */
+	install_test_bundle(fixture, user_data);
+
+	/* Manual partial cleanup */
+	test_umount(fixture->tmpdir, "mnt");
+	test_rm_tree(fixture->tmpdir, "mnt");
```

Comment:
`install_test_bundle()` should already handle the umount. Why is this needed?

Reply:
In fact, only the removal of the folder is needed. But this could be done by `install_test_bundle()` itself.